### DISCUSSION TITLE
UniFi Protect Version is not supported by this Integration

### DISCUSSION THREAD

**Discussion by ghost** [briis/unifiprotect#399] [Q&A] [answered]
When i try to configure unifi protect, i get the error message "UniFi Protect Version is not supported by this Integration"

It is a UniFi Cloud Key Gen2 Plus, SW: UnifiOS Version: 2.2.12
Unifi Protect version: 0.11-beta3
Ha Version: 2021.12.1

**Comment by AngellusMortis** [comment, marked as answer]
What version of UniFi Protect?

Check the pinned issue: https://github.com/briis/unifiprotect/issues/365

**Comment by ghost** [comment]
The problem was the unifi protect version on the cloud key, so today i learned something new :)
